app/models/rule.rb

```diff
@@ -70,10 +70,8 @@ def canonical?
     (profiles & Profile.canonical).any?
   end
 
-  def self.from_openscap_parser(op_rule, benchmark_id: nil, precedence: nil)
-    rule = find_or_initialize_by(ref_id: op_rule.id,
-                                 precedence: precedence,
```

Comment:
Maybe this piece was the one creating the problem. `precedence` should have been declared with `create_with`. WDYT?